index.html

```diff
@@ -30,6 +30,18 @@
                      <li class="desktop-Nav-Li"><a href="#">Contact</a></li>
                 </ul>
             </nav>
+            <div class="default"> 
+                <div class="mobile-menu-header">
+                    <img class="x-icon" src="images/IconCancel.svg" alt="xicon">
+                </div>
+                <div> 
+                <ul class="mobile-menu-ul">
+                <li class="mobile-menu-li"><a href="#Portfolio">Portfolio</a></li>
+                <li class="mobile-menu-li"><a href="#About">About</a></li>
+                <li class="mobile-menu-li"><a href="#Contact">Contact</a></li>
+                </ul>
```

Comment:
- [ ] Please make sure that the mobile nav takes the full width as provided in Figma, currently you have some white space on the left side
- [ ] Please make sure to give some space on the top of `X` button

Your design  | Figma design
------------- | -------------
<img width="320" alt="Screenshot 2022-03-01 at 20 00 50" src="https://user-images.githubusercontent.com/77038610/156203493-e48d328f-9f9f-44a8-b1c8-d841b8c46299.png"> | <img width="316" alt="Screenshot 2022-03-01 at 20 02 08" src="https://user-images.githubusercontent.com/77038610/156203718-4d62a005-3050-40ef-b8ee-ef536b6bbfa4.png">

> Here is the reason of the padding on the left 😉 
```
.navBar {
...
    padding: 0 2rem 0 1rem;
...
}
```

Reply:
All comments resolved @ShoiraTa